recipes/virtual-libjpeg/all/conanfile.py

```diff
@@ -0,0 +1,20 @@
+from conans import ConanFile, tools
+
+
+class VirtualjpegConan(ConanFile):
+    name = "virtual-libjpeg"
```

Comment:
I think a better name would be just `jpeg`

Reply:
We have:
- openjpeg
- libjpeg
- libjpeg-turboe
- mozjpeg
- openjpeg

I think jpeg is too generic.

Reply:
* mozjpeg: It's compatible with libjpeg API and ABI, and can be used as a drop-in replacement for libjpeg https://github.com/mozilla/mozjpeg#mozilla-jpeg-encoder-project-

If mozjpeg is another drop in replacement for libjpeg und libjpeg-turbo it should be added as an option too

Openjpeg seems to be it's own thing indeed (two times in your list btw)

I guess we can argue about `libjpeg` or `jpeg`, maybe `libjpeg` is actually better, but I think it is weird to have `virtual-` in it's name. We don't add `system-` to the name for system packages either

Reply:
"Virtual package" should be used by package creator over "real package". 
Adding "virtual-" in front of the name allow to quickly find virtual package by doing `conan search virtual-*`.

Reply:
openjpeg and mozjpeg added

Reply:
openjpeg doesn't provide libjpeg API